lib/getPlantUMLDiagram.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generates plantUML class diagram from default output syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default output syntax
+ * @returns {String} class diagram following plantUML syntax
+ */
+function getPlantUMLDiagram(metrics) {
+  let i=1;
+  let plantUMLSyntax = '';
+  metrics.forEach((relations,server) => {
```

Comment:
@derberg Since the first iteration is by the no. of different servers, in classDiagram we are using `server${i}` to match the relations and updating `i` by 1 after each iteration coz it would be a different server.

Reply:
o shit, I'm blind 😄 sorry